src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -50,6 +51,12 @@ internal class CircuitHost : IAsyncDisposable
             ILogger logger)
         {
             CircuitId = circuitId ?? throw new ArgumentNullException(nameof(circuitId));
+            CircuitSecret = circuitSecret;
+            if (circuitSecret.Secret is null)
```

Comment:
Are we adopting this as the new practice for checking for null? I've seen this being discussed as  a better option that avoids '==' overloading and other stuff. I'm fine if we adopt this style.

Reply:
Yes because VS now autocompletes this. There's no sense in fighting the tools.